cursor.go

```diff
@@ -166,8 +167,15 @@ func (c *Cursor) Fetch(db SqlPreparer) (batch *RowBatch, pkpos uint64, err error
 		return
 	}
 
+	// With the inline verifier, the columns to be selected may be very large as
+	// the query generated will be very large. The code here simply hides the
+	// columns from the logger to not spam the logs.
+
+	splitQuery := strings.Split(query, "FROM")
+	loggedQuery := fmt.Sprintf("SELECT [omitted] FROM %s", splitQuery[1])
```

Comment:
Can we add a simple test for this? Also, a nitpick, but maybe `omitted-md5-string` instead of just `omitted`?

Reply:
It's not just md5-string, there could also be other columns in there. I don't think it's necessarily worth to rebuild the other columns and omit only the md5 string tho.

Reply:
Right, `omitted-md5-transform` then. Just something a bit more descriptive.